config.conf.example

```diff
@@ -2,12 +2,13 @@
 HTML_URL = https://www.apertium.org
 APY_URL = https://www.apertium.org/apy
 
-#SUBTITLE =
-#SUBTITLE_COLOR =
+SUBTITLE
```

Comment:
Why was this changed? The previous construction was clearer.

Reply:
At first, I moved the fallback into the custom format file. But when I turned it into a conf file, I set the initial values directly to the fallbacks. I will move the non-`None` fallbacks back into `read-conf.py`, a simple `dict` should do.